addons/dragon/data/M47mis.rvmat

```diff
@@ -0,0 +1,81 @@
+class StageTI {
```

Comment:
From the `mis` suffix I deduce that this is the missle? does it really need a rvmat?

Reply:
Honestly I didn't look at anything major regarding the rvmat's or the models. After looking at it the actual missile model doesnt use the rvmat but rather the internal missiles for the static and hand-held model. I personally think its better with it